app-editors/featherpad/featherpad-0.18.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg qmake-utils
+
+MY_P="${P/featherpad/FeatherPad}"
+
+DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
+HOMEPAGE="https://github.com/tsujan/FeatherPad"
+SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
Looking at the source files, this seems to be `GPL-3+`.

Reply:
You are 100% correct. I just looked at the COPYING file, not the actual source code.

Reply:
It's a common issue, the license file doesn't really say which it is. Good to check source files for headers either way, they take precedence over these files in case of other licenses too.